app/controllers/providers/proceeding_merits_task/does_client_have_parental_responsibilities_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Providers
+  module ProceedingMeritsTask
+    class DoesClientHaveParentalResponsibilitiesController < ProviderBaseController
+      def show
+        @form = Providers::ProceedingMeritsTask::ParentalResponsibilitiesForm.new(model: proceeding)
+      end
+
+      def update
+        @form = Providers::ProceedingMeritsTask::ParentalResponsibilitiesForm.new(form_params)
+        # TODO: change the following to redirect to question 3 when it is added
+        return redirect_to providers_legal_aid_application_merits_task_list_path(legal_aid_application) if @form.relationship_to_child.eql?("false")
```

Comment:
This may need a draft suffix too
```suggestion
        return redirect_to providers_legal_aid_application_merits_task_list_path(legal_aid_application) if @form.relationship_to_child.eql?("false") && !draft_selected?
```